README.md

```diff
@@ -1,54 +1,88 @@
-# Kwil DB
+# KwilDB
```

Comment:
Sounds good to me, but there's not a single instance of "KwilDB" in https://docs.kwil.com/.

I've been unsure how we want to refer to our product.

Reply:
We've always referred to it as Kwil. I think I'll just change it to that